_posts/2016-05-30-one-model-to-rule-them-all.md

```diff
@@ -0,0 +1,154 @@
+---
+layout:      post
+title:       "One model to rule them all: cross-platform mobile test automation at BlaBlaCar"
```

Comment:
Yeah, I noticed they were all written with the French grammar rule. ;) I think it's something to review going forward, but I'd like the capitalization for this one to be updated, if there are no objections.